coreservices/iam/themes/catenax-central/email/messages/messages_en.properties

```diff
@@ -1,35 +1,7 @@
-{\rtf1\ansi\ansicpg1252\cocoartf2513
-\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;\f1\fnil\fcharset0 .SFNSMono-Regular;\f2\froman\fcharset0 Times-Roman;
-\f3\fnil\fcharset0 HelveticaNeue;\f4\fswiss\fcharset0 ArialMT;}
-{\colortbl;\red255\green255\blue255;\red189\green198\blue208;\red0\green0\blue0;\red26\green28\blue31;
-\red255\green255\blue255;\red175\green180\blue186;\red24\green25\blue27;}
-{\*\expandedcolortbl;;\cssrgb\c78824\c81961\c85098;\cssrgb\c0\c0\c0;\cssrgb\c13725\c14902\c16078;
-\cssrgb\c100000\c100000\c100000;\cssrgb\c74118\c75686\c77647;\cssrgb\c12549\c12941\c14118;}
-\paperw11900\paperh16840\margl1440\margr1440\vieww18600\viewh15320\viewkind0
-\pard\tx720\tx1440\tx2160\tx2880\tx3600\tx4320\tx5040\tx5760\tx6480\tx7200\tx7920\tx8640\pardirnatural\partightenfactor0
-
-\f0\fs24 \cf0 \
-\
-\
-\pard\pardeftab720\sl380\partightenfactor0
-
-\f1\fs27\fsmilli13600 \cf2 \expnd0\expndtw0\kerning0
-passwordResetSubject=CX Password Reset\
-passwordResetBody=Reset password link: \{0\}\
-passwordResetBodyHtml=<a href="\{0\}">
-\f2\fs24 \cf3 Dear 
-\f3\fs30 \cf4 \cb5 \outl0\strokewidth0 \strokec4 \{3\}
-\f2\fs24 \cf3 \cb1 \outl0\strokewidth0 ,
-\f4\fs32 \cf6 \cb7 <br />
-\f2\fs24 \cf3 \cb1 \
-\pard\pardeftab720\partightenfactor0
-\cf3 we have received your request to reset your password.
-\f4\fs32 \cf6 \cb7 <br />
-\f2\fs24 \cf3 \cb1 \
-By clicking on the button below, you will get redirected to the Catena-X password reset page. The password reset will be open for the next xx hours. Please proceed with your password request within the time.\
-In case your password is not getting reset in time, your account will be disabled until a password reset has took place.
-\f4\fs32 \cf6 \cb7 <br />
-\f2\fs24 \cf3 \cb1 \
-In case you havn\'92t initiated a password reset, please contact us immeditially by eplying on this email.
-\f1\fs27\fsmilli13600 \cf2 /a>\
-}
+passwordResetSubject=CX Password Reset
+passwordResetBody=Reset password link: {0}
+passwordResetBodyHtml=<a href="{0}">Dear {3},<br />
```

Comment:
instead of working with <br /> breaks I would recommend to structure the text in paragraphs:
`<p>Dear {3},</p>`
`<p>we have ...</p>`
`...`

Reply:
thanks thats a good idea, I will adjust it as soon as I know how i can change it :D